src/ButtonCatalog.js

```diff
@@ -22,6 +23,7 @@ export class ButtonHero extends Component {
   constructor(props) {
     super(props);
     this.ripples = [];
+    this.clickEvent = (el) => ReactGA.event({category: 'Component Interaction', action: 'button_click', label: el.target.textContent.trim()});
```

Comment:
Do you want to put `Component Interaction` and `button_click` into a constants file?